packages/hash/frontend/src/shared/ui/menu-item.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  listItemIconClasses,
+  listItemTextClasses,
+  MenuItem as MuiMenuItem,
+  MenuItemProps as MuiMenuItemProps,
+  typographyClasses,
+} from "@mui/material";
+import { FC, forwardRef, ReactNode } from "react";
+import { Link } from "./link";
+
+export type MenuItemProps = {
+  children?: ReactNode;
```

Comment:
Thanks. @kachkaev mentioned in an earlier PR that React 18 doesn't have children in React 18 typings and so we should probably use `VFC` everywhere. I should have used `VFC` instead of `FC` here. I missed that. Thanks for catching

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56210
https://solverfox.dev/writing/no-implicit-children/

Reply:
Weird how the article doesn't mention VFC – thanks for sharing this, I didn't know